src/App.tsx

```diff
@@ -1,25 +1,120 @@
 import './App.scss';
-
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import React, { useState } from 'react';
+import classNames from 'classnames';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
 
 export const App = () => {
+  const [newToDo, setNewToDo] = useState({
+    title: '',
+    user: '',
+  });
+  const todosCopy = [...todosFromServer];
+  const copy = [...usersFromServer];
```

Comment:
I don't see any sense to make two identical copies of same array

Reply:
it is different arrays, one of them is about todos, another is about the users )